### PR TITLE
Rename event handlers to the entity they are mapping

### PR DESCRIPTION
**Pull request by polymath-eric**

### CODE REVIEW

File: src/mappings/entities/mapExternalAgent.ts [hunk 1]
```diff
@@ -4,7 +4,10 @@ import { TickerExternalAgentAction, Sto } from "../../types";
 import { getOrDefault, serializeTicker } from "../util";
 import { EventIdEnum, ModuleIdEnum } from "./common";
 
-export async function handleTickerExternalAgentAction(
```
Comment (raycar5) [inline src/mappings/entities/mapExternalAgent.ts:7]: The entity is a TickerExternalAgentAction

File: src/mappings/entities/mapStaking.ts [hunk 2]
```diff
@@ -19,8 +19,10 @@ const bondedUnbondedOrReward = new Set([
   StakingEventType.Unbonded,
   StakingEventType.Reward,
 ]);
-
-export async function handleStakingEvent(
+/**
+ * Subscribes to events related to staking events
+ */
+export async function mapStaking(
```
Comment (raycar5) [inline src/mappings/entities/mapStaking.ts:25]: The entity is a StakingEvent

**Review by raycar5 — CHANGES_REQUESTED** [review]
I'm fine with changing it from `handle` to `map` but my mappers were already named after the entities they represent, not the event.

Also don't forget to rename the Funding entity in the` schema.graphql`

File: src/mappings/entities/mapStaking.ts (hunk 2 above)
Comment (polymath-eric) [reply src/mappings/entities/mapStaking.ts]: My bad, when I was doing funding I saw these and though I was supposed to add Event to the name.

Had another look at tooling and see my mistake now.

**Review by raycar5 — APPROVED** [review]

**Pull request merged**